server/liveblog/syndication/tasks.py

```diff
@@ -69,3 +70,31 @@ def fetch_image(self, url, mimetype):
         return FileStorage(stream=fetch_url(url), content_type=mimetype)
     except DownloadError as e:
         raise self.retry(exc=e, max_retries=SYNDICATION_CELERY_MAX_RETRIES, countdown=SYNDICATION_CELERY_COUNTDOWN)
+
+
+@celery.task(bind=True)
+def check_webhook_status(self, consumer_id):
+    """Check if consumer webhook is enabled by sending a fake http api request."""
+    from .utils import send_api_request
+    consumers = get_resource_service('consumers')
+    consumer = consumers._get_consumer(consumer_id)
+    if 'webhook_url' in consumer:
+        try:
+            response = send_api_request(consumer['webhook_url'], method='GET', json_loads=False)
+        except APIConnectionError as e:
+            raise self.retry(exc=e, max_retries=SYNDICATION_CELERY_MAX_RETRIES, countdown=SYNDICATION_CELERY_COUNTDOWN)
+        else:
+            # We can get "Authorization failed", as we are not providing an API Key.
+            # webhook_enabled field is updated using mongodb cursor to prevent multiple task execution
+            # on update.
+            cursor = consumers._cursor()
+            if response.status_code == 401:
+                webhook_enabled = True
+            else:
+                webhook_enabled = False
+
+            cursor.find_one_and_update({'_id': consumer['_id']}, {'$set': {'webhook_enabled': webhook_enabled}})
+            push_notification(consumers.notification_key, consumer={
```

Comment:
This notification needs to be handled by the client in order to display and update webhook_enabled status field.